exercises/somma.py

```diff
@@ -0,0 +1,9 @@
+def somma1(a,b)->int:
+    res=a+b
+    return res
```

Comment:
Puoi direttamente effettuare `return  a + b ` senza salvarti il valore in una variabile.